detector_test.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 //go:generate mockgen -package testmock -destination testmock/resolver.go github.com/buildpacks/lifecycle Resolver
+//go:generate mockgen -package testmock -destination testmock/layer_analyzer.go github.com/buildpacks/lifecycle LayerAnalyzer
```

Comment:
Does this belong here?

Reply:
I'll be honest, I only put it here because the other one was there. Where _should_ this go? I haven't used `mockgen` before this project. Should I pop this on `restorer_test.go` instead?